go/vt/mysqlctl/backup.go

```diff
@@ -44,6 +44,9 @@ const (
 
 	// the manifest file name
 	backupManifest = "MANIFEST"
+	// RestoreState is the name of the sentinel file used to detect whether a previous restore
+	// terminated abnormally
+	RestoreState = ".restore_in_progress"
```

Comment:
Could we make this a non-hidden file? If a human goes looking in that dir, I think it's useful for them to see the file so they know that a restore is in progress or has been interrupted. That could help them understand whatever issue they're trying to debug.